pkg/cli/webhook.go

```diff
@@ -87,8 +90,19 @@ func (c *WebhookServerCommand) RunUnstarted(ctx context.Context, args []string)
 	}
 	logger.Debugw("loaded configuration", "config", c.cfg)
 
-	pubsubClientOpts := append([]option.ClientOption{option.WithUserAgent("abcxyz/github-metrics-aggregator")}, c.testPubSubClientOptions...)
-	webhookServer, err := webhook.NewServer(ctx, c.cfg, pubsubClientOpts...)
+	agent := fmt.Sprintf("abcxyz/github-metrics-aggregator/%s", version.Version)
```

Comment:
```suggestion
	agent := fmt.Sprintf("abcxyz:github-metrics-aggregator/%s", version.Version)
```

The format for user-agent strings is structured and there can only be a single slash separating the name and version.